vsphere_storage_plugin/cidata.py

```diff
@@ -41,7 +41,9 @@ def create(rawvolume_client,
            volume_prefix,
            raw_files=None,
            **_):
-
+    if "cloudify.nodes.vsphere.CloudInitISO" in ctx.node.type:
```

Comment:
This is the opposite.
`cloudify.vsphere.*` is deprecated. `cloudify.nodes.vsphere.*` is desired.